Postman Test Suite/Record Ownership Test/Record Ownership Manual Setup Scripts.sql

```diff
@@ -0,0 +1,82 @@
+  --Client 1 should be able to full CRUD their School 1.
+  --Derived Resource -Client 1 has full CRUD - School 1
+  --Client 2 should be able to full CRUD School 2.
+  --Derived Resource -Client 2 has full CRUD - School 2
+  --Assuming Client 2 is assigned the extra ownership token, it should also be able to full CRUD School 1. 
+  --But Client 1, without the assignment to Ownership Token 2, should not be able to CRUD School 2.
+
+use EdFi_Admin_Test
+GO
+
+INSERT INTO [dbo].[OwnershipTokens] ([Description])     VALUES  ('OwnershipToken_One')
+INSERT INTO [dbo].[OwnershipTokens] ([Description])     VALUES  ('OwnershipToken_Two')
+
+DECLARE @ApiClient_ApiClientId INT;
+DECLARE @OwnershipToken_OwnershipTokenId INT;
+SELECT @ApiClient_ApiClientId = ApiClientId FROM [dbo].[ApiClients] where Name in ('255901001')
+
+SELECT @OwnershipToken_OwnershipTokenId = OwnershipTokenId FROM [dbo].[OwnershipTokens] where [Description] in ('OwnershipToken_One')
+INSERT INTO [dbo].[ApiClientOwnershipTokens]([ApiClient_ApiClientId],[OwnershipToken_OwnershipTokenId]) VALUES (@ApiClient_ApiClientId,@OwnershipToken_OwnershipTokenId)
+
+UPDATE [dbo].[ApiClients]
+   SET CreatorOwnershipTokenId_OwnershipTokenId = @OwnershipToken_OwnershipTokenId
+ WHERE ApiClientId =@ApiClient_ApiClientId
+
+
+SELECT @ApiClient_ApiClientId = ApiClientId FROM [dbo].[ApiClients] where Name in ('255901044')
```

Comment:
It's best not to reuse variables like this. It increases maintenance costs and likelihood of bugs in the future. Declare new variables for each value, set them once, and then don't change them.

Reply:
Fixed it .